src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int currentIndex = 0;
+
+        printHelloStatement();
+        while (true) {
+            String input;
+            input = in.nextLine();
+            if (input.equals("bye")) {
+                printByeStatement();
+                break;
+            } else if (input.equals("list")) {
+                printAllTasks(tasks);
+            } else if (input.startsWith("mark")) {
+                String[] temp = input.split(" ", 2);
+                int taskIndex = Integer.parseInt(temp[1]);
+                Task curTask = tasks[taskIndex - 1];
+                curTask.markAsDone();
```

Comment:
Would it be good to also extract these as a method? Likewise, for the below blocks too.